pytorch_blade/tests/disc/pdl/test_e2e/test_quantization.py

```diff
@@ -306,9 +254,13 @@ def forward(self, x):
         model = Model().eval().to(self.device)
         inp = torch.randn(8, 512, 512).to(self.device)
         traced_model = torch.jit.trace(model, inp)
+        if torch.version.cuda == '11.3':
```

Comment:
better to use range check， instead of a specific version.

Reply:
Have skiped version not in [11.3, 11.7] and added comments.